docs/UserGuide.md

```diff
@@ -104,11 +104,11 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
-### Locating persons by name: `find`
+### Find persons by keywords: `search`
 
-Finds persons whose names contain any of the given keywords.
+Finds all contacts that contain the KEYWORD(s)
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `search KEYWORD [MORE_KEYWORDS]`
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
```

Comment:
I checked again, and I think you missed out the find sales history feature under sales tracking.

Reply:
Yes, I was not sure about the format so I didn't update it.